internal/b/blueprints.go

```diff
@@ -91,10 +91,11 @@ type ApplicationService struct {
 }
 
 type Event struct {
-	Type     string
-	Sender   string
-	StateKey *string
-	Content  map[string]interface{}
+	Type          string
+	Sender        string
+	StateKey      *string
+	Content       map[string]interface{}
+	TransactionID *string
```

Comment:
I dislike this being here because it falls into the same area as the fields below:

> The following fields are ignored in blueprints as clients are unable to set them.

Really we should have an embedded struct to cleanly separate `b.Event` (i.e blueprint events) and client event e.g `client.Event` - the former has very few customisable fields due to it being run via the instruction runner, but the latter has many more fields to adjust.